.github/workflows/CI-DOCS.yml

```
@@ -0,0 +1,86 @@
+permissions: {}
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    tags: ["v*"]
+  pull_request:
+    types: [opened, reopened, ready_for_review]
+    branches: ["feature-*", "patch-*"]
+  pull_request_target:
+    types: [opened, edited, reopened, ready_for_review]
+    branches: ["main", "master", "stable", "HOTFIX-*"]
+    tags: ["v*"]
```

actionlint (1.7.4)

"tags" filter is not available for pull_request_target event. it is only for push event

(events)